File: exercicio - (9)pas ✔/e9.pas
program inicio;

var
  mtz: array[0..4, 0..4] of integer;
  sm: integer;
  i, j: integer;

begin

  for i := 0 to 4 do
    for j := 0 to 4 do
    begin
      write('Digite um número: ');
      readln(mtz[i, j]);
    end;

  sm := 0;
  for i := 0 to 4 do
    for j := i + 1 to 4 do
      if j > i then
        sm := sm + mtz[i, j];

  writeln('Soma dos elementos acima da diagonal principal: ', sm);
end.

File: e9.py
def inicio():
    mt = [[0 for _ in range(5)] for _ in range(5)]  
    sm = 0

    for i in range(5):
        for j in range(5):
            print('digite um número: ')
            mt[i][j] = int(input())
        
    sm = 0
    for i in range(5):
        for j in range(5):
            if j > i: 
                sm += mt[i][j]
        
    print("Soma dos elementos acima da diagonal principal: ", sm)

inicio()